src/Chapter4.hs

```diff
@@ -611,9 +650,21 @@ Implement the 'Monad' instance for our lists.
   maybe a few) to flatten lists of lists to a single list.
 -}
 
-
-{- |
-=💣= Task 8*: Before the Final Boss
+instance Monad List where
+  (>>=) :: List a -> (a -> List b) -> List b
+  as >>= afunc = 
+    let
+      -- we can extrat this as it is for applicative and can be reused
+      appa :: List a -> List a -> List a
+      appa Empty as = as
+      appa (Cons a as) bs = Cons a (appa as bs) 
```

Comment:
Totally correct 👍🏻 You can implement a top-level function `append` and reuse it in both `Applicative` and `Monad` instance